kvbc/include/merkle_tree_block.h

```diff
@@ -12,29 +12,25 @@
 #include <iterator>
 #include <unordered_map>
 
-namespace concord {
-namespace storage {
-namespace blockchain {
-namespace v2MerkleTree {
-namespace block {
+namespace concord::kvbc::v2MerkleTree::block {
 
 inline constexpr auto BLOCK_DIGEST_SIZE = bftEngine::SimpleBlockchainStateTransfer::BLOCK_DIGEST_SIZE;
 
 // Creates a block that includes a set of key/values. The passed parentDigest buffer must be of size BLOCK_DIGEST_SIZE
 // bytes.
 
-concordUtils::Sliver create(const concord::kvbc::SetOfKeyValuePairs &updates,
+concordUtils::Sliver create(const SetOfKeyValuePairs &updates,
                             const void *parentDigest,
                             const sparse_merkle::Hash &stateHash);
 
 // Returns the block data in the form of a set of key/value pairs.
-concord::kvbc::SetOfKeyValuePairs getData(const concordUtils::Sliver &block);
+SetOfKeyValuePairs getData(const RawBlock &block);
 
 // Returns the parent digest of size BLOCK_DIGEST_SIZE bytes.
-const void *getParentDigest(const concordUtils::Sliver &block);
+const void *getParentDigest(const RawBlock &block);
```

Comment:
I know it's not you but why is it void*?

Reply:
The idea is to just treat as a buffer. We should wrap it in some proper type.